docs/declarative-customization/view-formatting.md

```diff
@@ -262,52 +556,363 @@ The following image shows a customized card in Gallery layout:
+                        'sp-field-borderAllRegular sp-field-borderAllSolid sp-css-borderColor-neutralSecondary'))))"
+          },
+          "children": [
+            {
+              "elmType": "img",
+              "attributes": {
+                "src": "=if(@group.fieldData == 'California', 'https://upload.wikimedia.org/wikipedia/commons/thumb/0/01/Flag_of_California.svg/1920px-Flag_of_California.svg.png',
+                    if(@group.fieldData == 'Chicago', 'https://upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Flag_of_Chicago%2C_Illinois.svg/1920px-Flag_of_Chicago%2C_Illinois.svg.png',
+                    if(@group.fieldData == 'New York', 'https://upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Flag_of_New_York_City.svg/2560px-Flag_of_New_York_City.svg.png',
+                    if(@group.fieldData == 'Seattle', 'https://upload.wikimedia.org/wikipedia/en/thumb/6/6d/Flag_of_Seattle.svg/1920px-Flag_of_Seattle.svg.png',
+                    if(@group.fieldData == 'Washington DC', 'https://upload.wikimedia.org/wikipedia/commons/thumb/d/d4/Flag_of_the_District_of_Columbia.svg/2560px-Flag_of_the_District_of_Columbia.svg.png',
+                    ''))))"
+              },
+              "style": {
+                "max-width": "24px",
+                "max-height": "24px",
+                "margin-top": "2px",
+                "border-radius": "2px"
+              }
+            },
+            {
+              "elmType": "div",
+              "children": [
+                {
+                  "elmType": "span",
+                  "style": {
+                    "padding": "5px 5px 5px 5px",
+                    "font-weight": "500"
+                  },
+                  "txtContent": {
+                    "operator": "+",
+                    "operands": [
+                      "",
+                      "@group.fieldData.displayValue"
+                    ]
+                  }
+                }
+              ]
+            },
+            {
+              "elmType": "div",
+              "children": [
+                {
+                  "elmType": "div",
+                  "style": {
+                    "display": "flex",
+                    "flex-direction": "row",
+                    "justify-content": "center"
+                  },
+                  "children": [
+                    {
+                      "elmType": "div",
+                      "txtContent": "='has ' + @group.count + if(@group.count > '1', ' employees', ' employee')",
+                      "style": {
+                        "padding": " ",
+                        "font-weight": "500"
+                      }
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+
+#### Example: Color coded group header with aggregates
+
+In the example below we have list with group headers formatted with group aggregates.

```

Comment:
In the example below we have list with formatted group headers formatted and group aggregates.

line equivalent space